about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="top-page">
+        <a href=""><img src="images/facebook-icon.svg" alt="facebook"></a>
+        <a href=""><img src="images/twitter-icon.svg"></a>
+        <a href="">Logout</a>
+        <a href="">Mypage</a>
+        <a href="">Korean</a>
+    </div>
+    <header>
+        <div class="hamburger">
+            <img src="images/hamburger-icon.svg" alt="menu">
+        </div>
+        <div class="top-right">
+            <div class="mo">
+                <a href="index.html"><img src="images/main-logo.svg" alt="logo"></a>
+            </div>
+        </div>
+        <nav class="nav">
+            <a href="about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">News</a>
+            <div class="campaign">
+                <a href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
+    <section class="home-pages">
+        <div class="home">
+            <h1>CREATIVE COMMONS</br>GLOBAL SUMMIT 2015</h1>
+        </div>
+        <div class="paragraphab">
+            <p>This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value of opennes and sharing, creating This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value<br> A posative change with people from over 80 countries is taking place in October, in Korea A posative change with people from over 80 countries is taking place in October, in Korea</p>
+        </div>
+        <div class="hom-foots">
+            <p>Please avoid doing anything. Here you can ask nothing</p>
+            <h4>rofl@gcgn.com</h4>
+        </div>
+    </section>
+    <section class="m-page">
+        <div class="primary">
+            <p>Creative Commons Global Summit 2015 Logo</p>
```

Comment:
- [ ] [OPTIONAL] It would be more semantically if you use `hx` tag instead of `<p/>` tag 👍🏼